modules/sdk-lib-mpc/test/unit/tss/ecdsa/dklsDsg.ts

```diff
@@ -0,0 +1,63 @@
+import { DklsDsg } from '../../../../src/tss/ecdsa-dkls';
+import * as fs from 'fs';
+import * as crypto from 'crypto';
+import should from 'should';
+
+describe('DKLS Dsg 2x3', function () {
+  it(`should create signatures`, async function () {
```

Comment:
Can you create a test matrix showing signing between each combination of 2/3 parties? 

- user, bitgo
- user, backup
etc

Reply:
fixed